internal/chart/flags.go

```diff
@@ -74,6 +72,33 @@ func AppendExternalRegistryFlags(flags map[string]interface{}, enableInternal bo
 	return flags
 }
 
+func AppendDefaultPresetFlags(flags map[string]interface{}, defaultBuildJobPreset, defaultRuntimePodPreset string) map[string]interface{} {
```

Comment:
It's little confusing for me, because this function has param `flags` which is modified (out parameter) and the same value is returned. Why?
I'm familiar with functions which return new value and keep unchanged param or functions with out param and without returned value (or with value of another type).

Reply:
OK, it's similar to nearest functions that we keep it unchanged.